cypress/e2e/UserStory1.cy.js

```diff
@@ -0,0 +1,27 @@
+// UserStory1 Test: See user profile and verify member name and thumbnail
```

Comment:
Would be interesting to actually change the thumbnail for instance. There's some example in compose.

Reply:
I included a language change